src/commands/transaction/construct_transaction/add_action_1/add_action/add_key/mod.rs

```diff
@@ -1,8 +1,8 @@
 use strum::{EnumDiscriminants, EnumIter, EnumMessage};
 
-mod access_key_type;
+pub mod access_key_type;
 mod use_manually_provided_seed_phrase;
```

Comment:
Let's add pub here as well to be on the same page with the surrounding code

Reply:
Done